grakn-graql/src/main/java/ai/grakn/graql/internal/pattern/AbstractPattern.java

```diff
@@ -24,6 +24,8 @@
 import ai.grakn.graql.admin.PatternAdmin;
 
 /**
+ * Abstract implementation of {@link PatternAdmin}.
```

Comment:
I think that is given away in the class header. Maybe state what this abstract class is used for. What common functionality it houses. Etc. . .